docs/topics/Gateway_Events.md

```diff
@@ -338,7 +266,75 @@ Sent by the client to indicate a presence or status update.
 }
 ```
 
-### Connecting and Resuming
```

Comment:
Was this header removed intentionally?

Reply:
yeah i didn't think it was really descriptive of the contents under it, nor consistent with how the rest of the page is laid out

Reply:
Makes sense 👍